src/map/guild.cpp

```diff
@@ -2066,20 +2096,17 @@ int guild_break(struct map_session_data *sd,char *name) {
  * from char-server.
  */
 void guild_castle_map_init(void) {
-	int num = db_size(castle_db);
```

Comment:
Please change this whole dirty function...
C++ iterate over it and build a vector with the ids that need to be requested.
Thank you!

Reply:
The function is using some packets sent to charserver and I have some issue with that. I can't fix the issues on my local. Can you commit the changes when you have time?

Reply:
Done! :)